howto/update/upgrade-anbox.md

```diff
@@ -138,6 +146,21 @@ In case a reboot of the machine is required, a status message will be shown. Whe
 
     juju run --wait=1m lxd/0 clear-notification
 
+If the LXD charm is deployed on a machine with a NVIDIA GPU installed, the NVIDIA drivers are held from being upgraded by default in case of downtime for all running containers due to either a manual upgrade or an [unattended-upgrade](https://wiki.debian.org/UnattendedUpgrades). The downside to this is that the machine may miss security updates for the NVIDIA drivers. To manually upgrade the NVIDIA drivers, you need to:
+1. Stop the all running containers
+2. Unhold the NVIDIA drivers to allow an upgrade:
+
+        sudo apt-mark unhold linux-modules-nvidia-* nvidia-* libnvidia-*
+
+3. Update OS:
+
+        sudo apt update && sudo apt upgrade -y
+
+4. Hold the NVIDIA drivers from being updated again:
+
+        sudo apt-mark hold linux-modules-nvidia-* nvidia-* libnvidia-*
+
+A Juju action that simplifies the manual upgrade for the NVIDIA drivers will be added in the future release.
```

Comment:
```suggestion
A Juju action that simplifies the manual upgrade for NVIDIA drivers will be added in a future release.
```

Reply:
Isn't it risky to write 'will be added' here? Shouldn't we be more careful with the wording? I think it would be wise to be more prudent:
```suggestion
A Juju action that simplifies the manual upgrade for the NVIDIA drivers may be added in the future release.
```

Reply:
It is not uncommon to have such statements that talk about future releases but we must be sure that it is a fix that is absolutely going to be implemented - something without which we wouldn't go further. If it is not that certain, then we shouldn't include it.